sessionctx/variable/session.go

```diff
@@ -402,6 +402,15 @@ type SessionVars struct {
 
 	// ReplicaRead is used for reading data from replicas, only follower is supported at this time.
 	ReplicaRead kv.ReplicaReadType
+
+	// StartTime is the query start time.
```

Comment:
All variables in `SessionVars` are session-level, so the comment should be `the start time of the last query`?